packages/interface/src/components/plugin-preview-menu/index.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * WordPress dependencies
+ */
+import { createSlotFill, MenuItem } from '@wordpress/components';
+
+export const { Fill, Slot } = createSlotFill( 'PluginPreviewMenu' );
+
+/**
+ * Defines an extensibility slot for the device preview dropdown.
+ *
+ * The `title` and `icon` are used to populate the Preview menu item.
+ *
+ * @param {Object}   props         Component properties.
+ * @param {string}   props.name    A unique name of the custom preview. Prefix with plugin name.
+ * @param {Function} props.onClick Menu item click handler
+ * @param {string}   props.title   Menu item title.
+ */
+const PluginPreviewMenu = ( { name, onClick, title, ...props } ) => (
+	<Fill>
+		<MenuItem
+			className="block-editor-post-preview__button-resize"
```

Comment:
This classname doesn't follow our guidelines, the component is within "interface" (which can be acceptable) but the classname is like a "block editor" one.